client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -488,11 +488,15 @@ class PluginMeta extends Component {
 					<PluginAutomatedTransfer plugin={ this.props.plugin } />
 				}
 
-				{ ( get( this.props.selectedSite, 'jetpack' ) || this.hasBusinessPlan() || this.isWpcomPreinstalled() ) &&
+				{ ( this.props.selectedSite &&
+					get( this.props.selectedSite, 'jetpack' ) || this.hasBusinessPlan() || this.isWpcomPreinstalled() ) &&
 					<div style={ { marginBottom: 16 } } />
 				}
 
-				{ ! get( this.props.selectedSite, 'jetpack' ) && ! this.hasBusinessPlan() && ! this.isWpcomPreinstalled() &&
+				{ ! this.props.selectedSite &&
```

Comment:
I believe you want the opposite of this, don't you? You want to check to make sure a site was selected, so you probably want to remove the bang.